app/js/views/reminders.jsx

```diff
@@ -67,7 +67,7 @@ export default class Reminders extends React.Component {
     const reminders = this.state.reminders;
     reminders.push({
       id: reminders.length,
-      recipient: reminder.users,
```

Comment:
I have a doubt here: what was "recipient" supposed to hold ? An array or a string ?